docker-compose.mariadb.yml

```diff
@@ -13,6 +13,12 @@ services:
     environment:
       MARIADB_ROOT_PASSWORD: ${DB_PASSWD:-prestashop}
       MARIADB_DATABASE: ${DB_NAME:-prestashop}
+    healthcheck:
```

Comment:
Woah, cool, I did not know about the `condition: service_healthy` feature

For anybody else who did not know 😄 https://stackoverflow.com/questions/42567475/docker-compose-check-if-mysql-connection-is-ready

So healthcheck.sh is doing the same thing we are doing with /tmp/wait-for-it.sh but out of the box?

Reply:
yes. Puts the liveness testing back in the original container rather than every dependent service.